rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -0,0 +1,136 @@
+    Args:
+        config(object): Test configuration
+    """
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    umgmt = UserMgmt()
+    ceph_conf = CephConfOp()
+    rgw_service = RGWService()
+    # preparing data
+    user_name = resource_op.create_users(no_of_users_to_create=1)[0]['user_id']
+    tenant = 'tenant'
+    tenant_user_info = umgmt.create_tenant_user(tenant_name=tenant,
+                                                user_id=user_name,
+                                                displayname=user_name)
+    user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_name)
+
+    hostname = socket.gethostname()
+    ip = socket.gethostbyname(hostname)
+    port = utils.get_radosgw_port_no()
+
+    ip_and_port = '%s:%s' % (ip, port)
+    s3_auth.do_auth(tenant_user_info, ip_and_port)
+
+    bucket_name = utils.gen_bucket_name_from_userid(user_name, rand_no=0)
+
+    #Create a bucket
+    s3cmd_reusable.create_bucket(bucket_name)
+    log.info('Bucket %s created' % bucket_name)
+
+    #Upload file to bucket
+    uploaded_file = s3cmd_reusable.upload_file(bucket_name)
+    log.info('Uploaded file %s to bucket %s' %(uploaded_file, bucket_name))
+
+    #Delete file from bucket
+    s3cmd_reusable.delete_file(bucket_name, uploaded_file)
+    log.info('Deleted file %s from bucket %s' %(uploaded_file, bucket_name))
+
+    #Delete bucket
+    s3cmd_reusable.delete_bucket(bucket_name)
+    log.info('Bucket %s deleted' % bucket_name)
+
+    # check for any crashes during the execution
+    crash_info=reusable.check_for_crash()
+    if crash_info:
+        raise TestExecError("ceph daemon crash found!")
+
+if __name__ == '__main__':
+
+    test_info = AddTestInfo('test swift user key gen')
+
+    try:
+        project_dir = os.path.abspath(os.path.join(__file__, "../../.."))
+        test_data_dir = 'test_data'
+        TEST_DATA_PATH = (os.path.join(project_dir, test_data_dir))
+        log.info('TEST_DATA_PATH: %s' % TEST_DATA_PATH)
+        if not os.path.exists(TEST_DATA_PATH):
+            log.info('test data dir not exists, creating.. ')
+            os.makedirs(TEST_DATA_PATH)
+        parser = argparse.ArgumentParser(description='RGW Swift Automation')
+        parser.add_argument('-c', dest="config",
+                            help='RGW Test yaml configuration')
+        parser.add_argument('-log_level', dest='log_level',
+                            help='Set Log Level [DEBUG, INFO, WARNING, ERROR, CRITICAL]',
+                            default='info')
+        args = parser.parse_args()
+        yaml_file = args.config
+        log_f_name = os.path.basename(os.path.splitext(yaml_file)[0])
+        configure_logging(f_name=log_f_name,
+                          set_level=args.log_level.upper())
+        config = resource_op.Config(yaml_file)
+        config.read()
```

Comment:
add `configure_logging()` functions here. to setup the loggin for this test case.

Reply:
It looks configure_logging function is already added in test file